pymc3/distributions/timeseries.py

```diff
@@ -275,7 +275,7 @@ def _random(self, sigma, mu, size, sample_shape):
         """Implement a Gaussian random walk as a cumulative sum of normals.
         axis = len(size) - 1 denotes the axis along which cumulative sum would be calculated.
         This might need to be corrected in future when issue #4010 is fixed.
-        Lines 318-322 ties the starting point of each instance of random walk to 0"
+        Lines 291-295 set the starting point of each instance of the random walk to 0."
```

Comment:
I would refer to the method name instead of lines which is going to be brittle.

Reply:
Good idea; I moved the comment from the docstring to the location in the code in 1761a2e.